cmd/basic_automations/main.go

```diff
@@ -0,0 +1,86 @@
+package main
+
+import (
+	"flag"
+	"github.com/blinkops/blink-sdk/library_automations/consts"
+	"github.com/blinkops/blink-sdk/library_automations/library/generator"
+	"github.com/blinkops/blink-sdk/library_automations/library/generator/actions"
+	"github.com/blinkops/blink-sdk/pkg/blink_io"
+	"github.com/blinkops/blink-sdk/plugin/config"
+	"github.com/pkg/errors"
+	log "github.com/sirupsen/logrus"
+	"io/fs"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+)
+
+func main() {
+	var err error
+
+	contentPath := flag.String("content-path", "", "the path that contains the content packs to be uploaded")
+	genPath := flag.String("gen-path", "", "the path that contains the generated integrations")
+	configPath := flag.String("config-path", "", "the path to the file which contains the black listed actions")
+
+	flag.Parse()
+
+	if err = os.Setenv(config.ConfigurationPathEnvVar, filepath.Join(*configPath, "config.yaml")); err != nil {
+		log.Fatal(errors.Wrap(err, "failed to set configuration env variable: "))
```

Comment:
```suggestion
		log.Fatal(errors.Wrap(err, "failed to set configuration env variable"))
```

errors.Wrap adds the : automatically